pkg/activator/handler/handler.go

```diff
@@ -133,8 +148,15 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		// Report the metrics
 		duration := time.Since(start)
 
-		a.Reporter.ReportRequestCount(namespace, ar.ServiceName, ar.ConfigurationName, name, httpStatus, attempts, 1.0)
-		a.Reporter.ReportResponseTime(namespace, ar.ServiceName, ar.ConfigurationName, name, httpStatus, duration)
+		var configurationName string
+		var serviceName string
+		if revision.Labels != nil {
+			configurationName = revision.Labels[serving.ConfigurationLabelKey]
+			serviceName = revision.Labels[serving.ServiceLabelKey]
+		}
+
+		a.Reporter.ReportRequestCount(namespace, serviceName, configurationName, name, httpStatus, attempts, 1.0)
```

Comment:
I know this was here before, but do we want to report even if the names are empty?
Since this seems like a rouge request anyway?